source/localizable/object-model/computed-properties.md

```diff
@@ -31,6 +31,20 @@ This declares the function to be a computed property, and the arguments tell Emb
 
 Whenever you access the `fullName` property, this function gets called, and it returns the value of the function, which simply calls `firstName` + `lastName`.
 
+When you want to depend on a property which belongs to an object, you can setup multiple dependent keys by using brace expansion:
+
+```javascript
+var obj = Ember.Object.extend({
+  baz: {foo: 'BLAMMO', bar: 'BLAZORZ'},
+
+  something: function(){
+    return this.get('baz.foo') + ' ' + this.get('baz.bar');
+  }.property('baz.{foo,bar}')
```

Comment:
@codeNgJon Looks great!  One thing we've been shifting toward is an updated syntax that would look more like this:

```
something: Ember.computed('baz.{foo,bar}') {
    return this.get('baz.foo') + ' ' + this.get('baz.bar');
}
```

Would you be able to test that that works (possibly on Ember Twiddle) and update this PR?